src/test/scala/is/hail/stats/SkatModelSuite.scala

```diff
@@ -0,0 +1,250 @@
+package is.hail.stats
+
+import is.hail.SparkSuite
+import is.hail.TestUtils._
+import is.hail.utils._
+import is.hail.check.Prop._
+import org.testng.annotations.Test
+import breeze.linalg._
+
+import scala.language.postfixOps
+import scala.sys.process._
+import java.io.File
+
+import breeze.numerics._
+import com.google.common.primitives.Doubles
+import is.hail.stats.SkatModelTemp
+import org.scalatest.testng.TestNGSuite
+
+class SkatModelSuite extends SparkSuite {
+
+  val header = "==============================================="
```

Comment:
In tests I think it's ok to keep print statements that are especially helpful but they should be commented out so the logs and terminal don't get crowded. We can ask Cotton what he prefers too.